democracy/views/hearing.py

```diff
@@ -62,23 +44,40 @@ def get_sections(self, hearing):
         serializer.bind('sections', self)  # this is needed to get context in the serializer
         return serializer.to_representation(queryset)
 
+    def get_main_image(self, hearing):
```

Comment:
Yes, but I'm thinking of a situation where there are two images, first one non public and second one public. In that case the returned main image would be a different image between normal user and superuser, which doesn't seem good.